### PR TITLE
Use IMDB datastore

### PR DESCRIPTION
**Pull request by tkyi**
Update aws-api to use new IMDB datastore
https://github.com/screwdriver-cd/screwdriver-datastore-imdb/pull/1

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -1,3 +1,17 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable consistent-return */
+
```
Comment (FenrirUnbound) [inline app/index.js]: Extra space?

Comment (FenrirUnbound) [inline app/index.js]: Or is it for readibility?

**Comment by FenrirUnbound** [comment]
👍 

Looks fine to me

jy960qt - imgur

**Comment by nkatzman** [comment]
👍

**Pull request merged**